### PR TITLE
zz in visual, visualline, and visual block mode

### PR DESCRIPTION
**Pull request by xconverge**
Yay! We love PRs! 🎊

Please include a description of your change and ensure:

- [x] Commit message has a short title & issue references
- [x] Each commit does a logical chunk of work.
- [x] It builds and tests pass (e.g `gulp`)

More info can be found on our [contribution guide](https://github.com/VSCodeVim/Vim/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: src/actions/actions.ts [hunk 1]
```diff
@@ -1873,6 +1874,22 @@ class CommandCenterScroll extends BaseCommand {
 }
 
 @RegisterAction
+class CommandCenterScrollMultipleLines extends BaseCommand {
+  modes = [ModeName.VisualLine, ModeName.VisualBlock];
+  keys = ["z", "z"];
+
+  public async exec(position: Position, vimState: VimState): Promise<VimState> {
+    // In these modes, you want to take the full selection and find the middle point
```
Comment (johnfn) [inline src/actions/actions.ts]: Are you sure? For me, zz just centers on the active cursor.

**Comment by johnfn** [comment]
I think it's sufficient to add all visual modes to the mode list for zz.

File: src/actions/actions.ts (hunk 1 above)
Comment (xconverge) [inline src/actions/actions.ts]: Apparently I made up functionality, I am not sure how I tested it but it was wrong. Thanks for carefully checking what I am actually doing....

**Comment by johnfn** [comment]
Good stuff! Thanks for your work, @xconverge!

**Pull request merged**